payment_term_type/README.md

```diff
@@ -0,0 +1,8 @@
+Payment Terms Type
+==================
+
+This module allow to show in payment terms the payment type depending of the lines to compute
+
+* Credit
+* Cash
```

Comment:
@hbto 
La correcta traducción para referirse a tipo de pago a crédito y a un tipo de pago de contado... es: credit and cash?
O cual otra palabra pudiera ser, sin se que pueda mal interpretar como pago con dinero en efectivo